### ISSUE TITLE
Controller Vibration Stopped Working

### ISSUE THREAD

**Issue by Wolfarix** [StarGazer1258/BeatDrop#219]
**Environment:**
Device - Ryzen 7 2700X, 32GB RAM, RTX 2070 Supper
OS - Windows 10, Version 2004, Build 19041.207
BeatDrop Version - BeatDrop Version 2.6.2
**Description of bug:**
When I was in the middle of playing a custom song the controller vibration stopped working and will not come back.

**Steps to reproduce:**
Played custom song and it suddenly stopped working

**Expected behavior:**
Controllers should vibrate every time you hit a block, bomb, barrier or saber.

**Actual Behavior**
No controller vibrations

**Comment by Jverm84** [comment]
I’ve had this exact same issue.

**Comment by Xonor13** [comment]
I don't believe this is an issue with BeatDrop. Sounds like a problem with the actual Beatsaber game.

**Comment by Wolfarix** [comment]
Well I’ve been on again off again with this client and every time the client is installed this problem happens. Unfortunately it seems that in order to make it happen it’s out of luck but I can say that this dose not happen when the client is not installed.
